requirements/pip.in

```diff
@@ -0,0 +1,5 @@
+# Core dependencies for installing other packages
+
+pip==20.0.2
```

Comment:
Why don't we use the latest pip version instead? as we are doing in other packages?

Reply:
pip 20.0.2 was being explicitly installed in travis config. Using the latest version raised `'AttributeError: 'ParsedRequirement' object has no attribute 'req'`